src/redux/store.js

```diff
@@ -0,0 +1,10 @@
+import { configureStore } from '@reduxjs/toolkit';
+import booksReducer from './books/booksSlice';
+
```

Comment:
- [ ] Please import `categoriesSlice` from `src/redux/categoriesSlice` as a reducer as you did for the books. It can look like this
```import categoriesReducer from './categories/categoriesSlice';```